### ISSUE TITLE
When I used pb and pbtxt the result was different from tensorflow.

### ISSUE THREAD

**Issue by vision-one** [opencv/opencv#19232]
OS : ubuntu 16.04 LTS
Opencv : 4.2.0 (with CUDA)
Tensorflow version : 1.13.2

I made a pb file by modifying threshold in "**second_stage_post_processing**" of resnet50 config through the tensorflow object detection api. 

    feature_extractor {
      type: 'faster_rcnn_resnet50'
      first_stage_features_stride: 16
    }
    first_stage_anchor_generator {
      grid_anchor_generator {
        scales: [0.25, 0.5, 1.0, 2.0]
        aspect_ratios: [0.5, 1.0, 2.0]
        height_stride: 16
        width_stride: 16
      }
    }
    first_stage_box_predictor_conv_hyperparams {
      op: CONV
      regularizer {
        l2_regularizer {
          weight: 0.0
        }
      }
      initializer {
        truncated_normal_initializer {
          stddev: 0.01
        }
      }
    }
    first_stage_nms_score_threshold: 0.0
    first_stage_nms_iou_threshold: 0.7
    first_stage_max_proposals: 300
    first_stage_localization_loss_weight: 2.0
    first_stage_objectness_loss_weight: 1.0
    initial_crop_size: 14
    maxpool_kernel_size: 2
    maxpool_stride: 2
    second_stage_box_predictor {
      mask_rcnn_box_predictor {
        use_dropout: false
        dropout_keep_probability: 1.0
        fc_hyperparams {
          op: FC
          regularizer {
            l2_regularizer {
              weight: 0.0
            }
          }
          initializer {
            variance_scaling_initializer {
              factor: 1.0
              uniform: true
              mode: FAN_AVG
            }
          }
        }
      }
    }
    second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.9   <<<<<<<<<<<<<<<<<<<<<<<<<
        iou_threshold: 0.9   <<<<<<<<<<<<<<<<<<<<<<<<<
        max_detections_per_class: 100
        max_total_detections: 300
      }
      score_converter: SOFTMAX
    }
    second_stage_localization_loss_weight: 2.0
    second_stage_classification_loss_weight: 1.0
  }

In TensorFlow, the inference is very good through the learned pb file.
But after applying "tf_text_graph_faster_rcnn.py", OPENCV dnn readNetFromTensorflow shows different results.

How can I apply the threshold value of second_stage_post_processing to pbtxt?

**Comment by dkurt** [comment]
You may open a `.pbtxt` file and find `nms_threshold` parameter.
